### PR TITLE
[51] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by Tocyn**
Task: `HTML FORMS`

Screen Shot 2020-10-07 at 8 17 58 PM

### CODE REVIEW

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:2]: No `<head>` and `<body>` tags

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Thank you @adaugochi.
I have added the tags.

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    <head>
+     <h1> Contact Form </h1>
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:4]: `<h1>` and `<form>` tags are not tags that can be found on the `<head>` section. Tags placed inside the `<head>` tag are `<meta>, <link>, <title>, <script>`. So, kindly correct yourself

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Thank you @adaugochi, I have made the required changes.

Warm Regards.

**Review by adaugochi — CHANGES_REQUESTED** [review]
I am still not seeing `<head>` and `<body>` tags

**Comment by Tocyn** [comment]
@adaugochi, I followed the slide example so I am confused on the head and
body tags you are referring to.

**Comment by Tocyn** [comment]
@adaugochi, I have effected some changes. Kindly review.

**Comment by Tocyn** [comment]
@ adaugochi, kindly review my PR. Thank you.

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <Form action = "#" method =post>
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:3]: Your `<form>` tag should be after closing `<head>` tag. Also, it should be all in lowercases

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      <input type="email" name="email" id="email" placeholder="enter your email address"><br>
+
+      <label for= "subject">Subject</Label><br>
+      <input type="text" name= "subject" id = "subject"> <br>
+
+      <label for="comment">Message Input</label><br/>
+      <textarea name="comments" id="comments"rows ="10" cols="30"> 
+      </textarea><br>
+
+      <Button type="submit">submit</Button>
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:25]: tags should be all in lowercases

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Thanks @adaugochi. I have effected required changes.
Kindly review.

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <meta> CONTACT FORM <br>
+    </head>
+    <form action = "#" method =post></form>
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:6]: This should be inside the `<body>` tag. And there is nothing within your `<form>` tag also, why?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <meta> CONTACT FORM <br>
+    </head>
+    <body>
+      <form action = "#" method =post entype="application/x-www-form-urlencoded"></form>
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:7]: Your `<form>` tag can not be empty. It should have some nested element within it

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
I have effected the requested changes, kindly review my PR.

**Review by adaugochi — CHANGES_REQUESTED** [review]
You haven't effect the changes yet.

Kindly remove the `contactformdata.html` file

**Comment by Tocyn** [comment]
I will remove the file link.
 I don't understand the other  changes you want me to effect, I have edited
to the best of my understanding.kindly explain further.

**Comment by Tocyn** [comment]
Good morning, kindly review my PR

File: submissions/oluwatosinOke/html-forms/contactform.html [hunk 7]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+      <meta> CONTACT FORM <br>
+    </head>
+    <body>
+      <form action = "#" method =post entype="application/x-www-form-urlencoded">
```
Comment (adaugochi) [inline submissions/oluwatosinOke/html-forms/contactform.html:7]: remove the attributes on this `<form>` tag. There are not necessary yet

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by Tocyn** [comment]
Attribute removed. Kindly review.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**